### PR TITLE
Adding option to allow for http communication to management api

### PR DESCRIPTION
**Pull request by Lileso**
I saw that HTTP had been requested for testing purposes here:  https://community.gravitee.io/t/helm-chart-apim3-issues/311/10 and thought I'd add this in.

**Issue**

https://github.com/gravitee-io/issues/issues/8957

**Description**

-  `ui-configmap`: adjusted so BaseURL option pulls from the values file whether to use HTTP or HTTPS
- `portal-configmap`: adjusted so BaseURL option pulls from the values file whether to use HTTP or HTTPS
- `readme.adoc`: Added documentation for the UI but couldn't find where the portal section was so didn't add docs for that one

### CODE REVIEW

**Review by gaetanmaisse — COMMENTED** [review]
Hey @Lileso, thanks for the contribution! 🎉 

@sebdevaux and I discussed the PR internally, and the main feedback is the naming of `apiMethod`, we think it would be better named with `scheme`. Can you update the PR accordingly?

**Comment by Lileso** [comment]
Oh yea that is a better name, I've updated the PR now.

File: apim/3.x/templates/ui/ui-configmap.yaml [hunk 1]
```diff
@@ -20,9 +20,9 @@ data:
       "baseURL": "{{ .Values.ui.baseURL }}"
       {{- else }}
       {{- if hasSuffix "/" .Values.api.ingress.management.path }}
-      "baseURL": "https://{{ index .Values.api.ingress.management.hosts 0 }}{{ .Values.api.ingress.management.path }}organizations/DEFAULT/environments/DEFAULT/"
+      "baseURL": "{{.Values.ui.scheme }}://{{ index .Values.api.ingress.management.hosts 0 }}{{ .Values.api.ingress.management.path }}organizations/DEFAULT/environments/DEFAULT/"
```
Comment (sebdevaux) [inline apim/3.x/templates/ui/ui-configmap.yaml:23]:
```suggestion
      "baseURL": "{{.Values.api.ingress.management.scheme }}://{{ index .Values.api.ingress.management.hosts 0 }}{{ .Values.api.ingress.management.path }}organizations/DEFAULT/environments/DEFAULT/"
```

File: apim/3.x/templates/ui/ui-configmap.yaml [hunk 2]
```diff
@@ -20,9 +20,9 @@ data:
...
       {{- else }}
-      "baseURL": "https://{{ index .Values.api.ingress.management.hosts 0 }}{{ .Values.api.ingress.management.path }}/organizations/DEFAULT/environments/DEFAULT/"
+      "baseURL": "{{.Values.ui.scheme }}://{{ index .Values.api.ingress.management.hosts 0 }}{{ .Values.api.ingress.management.path }}/organizations/DEFAULT/environments/DEFAULT/"
```
Comment (sebdevaux) [inline apim/3.x/templates/ui/ui-configmap.yaml:25]:
```suggestion
      "baseURL": "{{.Values.api.ingress.management.scheme }}://{{ index .Values.api.ingress.management.hosts 0 }}{{ .Values.api.ingress.management.path }}/organizations/DEFAULT/environments/DEFAULT/"
```

File: apim/3.x/values.yaml [hunk 3]
```diff
@@ -1025,6 +1025,7 @@ gateway:
 portal:
   enabled: true
   name: portal
+  scheme: https # should be https or http
```
Comment (sebdevaux) [inline apim/3.x/values.yaml:1030]:
```suggestion
```

File: apim/3.x/values.yaml [hunk 4]
```diff
@@ -1170,6 +1171,7 @@ ui:
   enabled: true
   name: ui
   companyName: Gravitee.io
+  scheme: https # https or http
```
Comment (sebdevaux) [inline apim/3.x/values.yaml:1176]:
```suggestion
```

File: apim/3.x/README.adoc [hunk 5]
```diff
@@ -339,6 +339,10 @@ Please be aware that the Elasticsearch installed by Gravitee is NOT recommended
 `+null+`, defaults to Management API ingress value)_
 |`+[apim.example.com]/management+`
 
+|`+ui.scheme+` |Whether to use HTTP or HTTPS to communicate with Management API,
```
Comment (sebdevaux) [inline apim/3.x/README.adoc:342]:
```suggestion
```

**Review by sebdevaux — CHANGES_REQUESTED** [review]

File: apim/3.x/README.adoc (hunk 5 above)
Comment (Lileso) [reply apim/3.x/README.adoc:342]: Just reading your latest commit, Do you need me to add docs to the readme file for?: 
- `api.ingress.management.scheme`
- `api.ingress.portal.scheme`

**Pull request merged**